<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 7: Disk Operations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Disk:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiskName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>labelDiskName</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total size: 360 Kb</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Free size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiskFreeSize">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>labelDiskFreeSize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateFile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>690</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteFile">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>690</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteAllFiles">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>690</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All Files</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableDiskView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1061</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
